components/Home/Services/ServicesCard/ServicesCard.tsx

```diff
@@ -0,0 +1,39 @@
+import ComposedTitles from "@components/Common/ComposedTitles";
+import PrimaryDescription from "@components/Common/PrimaryDescription";
+import ServicesSvg from "@components/Common/ServicesSvg";
+import { ServicesInfo } from "@lib/types";
+
+interface ServicesCardProps {
+  data: ServicesInfo;
+  index: number;
+}
+const ServicesCard = ({ data, index }: ServicesCardProps) => {
+  const { id, title, description } = data;
+  const printSvgTop = () => {
+    if (id == 1) {
+      return <ServicesSvg />;
+    }
+    return "";
+  };
+  const printSvgBottom = () => {
+    if (id == 3) {
```

Comment:
@oscarcorellan For equality operators, always use the triple equals operator `===`. Using double equals `==` can generate strange behaviors and often generate false positive results, or viceversa. That's because JavaScript will try to convert the type of the values that you're comparing. You can learn more [here](https://www.freecodecamp.org/news/javascript-triple-equals-sign-vs-double-equals-sign-comparison-operators-explained-with-examples/#:~:text=Double%20Equals%20(%20%3D%3D%20)%20checks%20for%20value%20equality%20only.&text=On%20the%20other%20hand%2C%20Triple,difference%20through%20a%20few%20examples.)

```suggestion
    if (id === 1) {
```